samples/MusicStore/ForTesting/Mocks/OpenIdConnect/OpenIdConnectBackChannelHttpHandler.cs

```diff
@@ -12,7 +13,12 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
             var response = new HttpResponseMessage();
 
             var basePath = Path.GetFullPath(Path.Combine(
-                Directory.GetCurrentDirectory(), "ForTesting", "Mocks", "OpenIdConnect"));
+#if NET451
```

Comment:
It's a bug. The vstest team is tracking a fix for this here: https://github.com/Microsoft/vstest/issues/311